Dockerfile

```
@@ -3,16 +3,16 @@ FROM alpine:3.15.0
 ARG TARGETOS
 ARG TARGETARCH
 
-RUN apk add --no-cache --upgrade && apk add --no-cache tini=0.19.0-r0 curl bind-tools
+RUN apk add --no-cache --upgrade && apk add --no-cache tini curl bind-tools
 
-ADD --chmod=755 bin/nri-kubernetes-${TARGETOS}-${TARGETARCH} /var/db/newrelic-infra/newrelic-integrations/bin/
+ADD bin/nri-kubernetes-${TARGETOS}-${TARGETARCH} /bin/
```

Why do we not add the file, rename it and change its mode in the same layer/step?
```suggestion
ADD --chmod=755 bin/nri-kubernetes-${TARGETOS}-${TARGETARCH} /bin/nri-kubernetes
```

---

To clarify on the cryptic comment: Last time we checked it, doing what you suggest caused the file to end up as:
```
/bin/nri-kubernetes/nri-kubernetes-linux-amd64
```
And it happened only on the CI environment, IIRC. But please do a PR, this behavior might have changed.

---

`DOCKER_BUILDKIT=1` do really weird things... I asked out of curiosity because I thought that should be the intended behavior.

Maybe there are even more differences between `ADD` and `COPY` than I expected.